pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -108,6 +118,23 @@ in
               jq --sort-keys "del(.. | .checkedAt?)" $f | sponge $f
             done
 
+            # Ensure consistent permissions
+            # NOTE: For reasons not yet fully understood, pnpm might create files with
+            # inconsistent permissions, for example inside the ubuntu-24.04
+            # github actions runner.
```

Comment:
This is probably related to the sandbox. What installed was used? The cachix one with the sanbox or another one? I think I've hit the same problem before, see https://github.com/NuschtOS/search/commit/2ab7909ac0430e456fad99fe92723ea2066f8139

Reply:
In https://github.com/NixOS/nixpkgs/pull/350063 `nix-quick-install` was used and permissions were 555 on a `ubuntu-24.04` runner, but permissions were 444 on `ubuntu-22.04` runner (and their local env)

I'm not sure which runner was used in https://github.com/NixOS/nixpkgs/issues/422889, cc @jfraudeau @jmoggr